pkg/core/dao/dao_test.go

```diff
@@ -176,15 +176,26 @@ func TestStoreAsTransaction(t *testing.T) {
 		tx.Signers = append(tx.Signers, transaction.Signer{})
 		tx.Scripts = append(tx.Scripts, transaction.Witness{})
 		hash := tx.Hash()
+		si := stackitem.NewArray([]stackitem.Item{stackitem.NewBool(false)})
+		argBytes, err := dao.GetItemCtx().Serialize(si, false)
+		require.NoError(t, err)
+		ci := state.NewContractInvocation(util.Uint160{}, "fakeMethodCall", argBytes, 1, false)
 		aer := &state.AppExecResult{
 			Container: hash,
 			Execution: state.Execution{
 				Trigger: trigger.Application,
-				Events:  []state.NotificationEvent{},
-				Stack:   []stackitem.Item{},
+				Events: []state.NotificationEvent{
+					{
+						ScriptHash: util.Uint160{},
+						Name:       "fakeTransferEvent",
+						Item:       si,
+					},
+				},
+				Stack:       []stackitem.Item{},
+				Invocations: []state.ContractInvocation{*ci},
 			},
 		}
-		err := dao.StoreAsTransaction(tx, 0, aer)
```

Comment:
Not a proper test case for that. This change is not related to `TestStoreAsTransaction/no_conflicts`.